Gemfile

```diff
@@ -122,7 +122,7 @@ group :test do
   gem 'climate_control', '~> 0.2'
   gem 'faker', '~> 3.1'
   gem 'json-schema', '~> 3.0'
-  gem 'rack-test', '~> 2.0'  
```

Comment:
Manually fixed this one, but opened #23543 to have the IDE do it automatically in the future